meerk40t/core/elements.py

```diff
@@ -5124,6 +5125,14 @@ def reload_file(node, **kwgs):
             node.remove_node()
             self.load(filepath)
 
+        @self.tree_operation(_("Open in System: '%s'") % "{name}", node_type="file", help="")
```

Comment:
May I suggest `@self.tree_operation(_("Open in System: '{name}'").format(name=name), node_type="file", help=_("Open this file in the system application associated with this type of file"))`

Reply:
The {name} would get replaced wrong there. It's actually using {name} in the tree to replace that with the file's name. In a mock sort of operation thing.

Reply:
Ah - ok. Does that mean it could be simplified to: `@self.tree_operation(_("Open in System: '{name}'"), node_type="file", help=_("Open this file in the system application associated with this type of file"))`?

Reply:
Yeah, and in fact it should. We're going to allowing translators to move those `.format()` stuff around. So this shouldn't be any different.